### PR TITLE
feat: add credential session creation api

### PR DESCRIPTION
**Pull request by salman-tw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,28 @@ Returns a `PaginatedResponse` containing an array of `Company`s.
...
+
+**Response**
+
+Returns a `PaginatedResponse` containing an array of `Company`s.
+
+**Example**
+
+```ts
+await client.companies.get({ query: 'USWNT' });
+```
```
Comment (estrattonbailey) [inline README.md:247]: Copy pasta?

File: lib/types.ts [hunk 2]
```diff
@@ -241,6 +248,14 @@ export type Target = {
   company?: Pick<Company, 'id'> | Pick<Company, 'name'> | Company;
   date_of_birth?: string;
 };
+export type CredentialsTarget = {
+  first_name: string;
+  last_name: string;
+  social_security_number: string;
+  contact_email: string;
+  date_of_birth: string;
```
Comment (estrattonbailey) [inline lib/types.ts:256]: I left DOB optional in that pydantic update yesterday. Which should it be?

File: README.md (hunk 1 above)
Comment (salman-tw) [reply README.md]: derp

File: lib/types.ts (hunk 2 above)
Comment (salman-tw) [reply lib/types.ts:256]: required for now until we have screens to collect it

**Review by estrattonbailey — APPROVED** [review]

**Pull request merged**